plugins/vuetify.ts

```diff
@@ -15,7 +15,7 @@ import { fab } from "@fortawesome/free-brands-svg-icons";
 
 import "@/assets/scss/style.scss";
 
-// @ts-ignore
+// @ts-expect-error missing types in vuetify
 import colors from "vuetify/lib/util/colors";
```

Comment:
use `import colors from 'vuetify/util/colors'`

Reply:
I will fix vuetify once feat-upload-2 is merged

Reply:
or `import { blue, green } from 'vuetify/util/colors'`

![imagen](https://github.com/bitsongofficial/studio/assets/6311119/f80f24e2-edb0-44ca-9b21-6fe6d50d4392)